### PR TITLE
[C-1601][C-1602] Fix splash screen and status bars on android

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

* Fixes splash screen on android (device consistency) and status bar colors.
Unfortunately, android compatibility seems to be difficult with our splash screen animation & the bootsplash library. It works well on pixel phones, but poorly on galaxy s10 (and I would guess other devices as well).
https://github.com/zoontek/react-native-bootsplash/issues/410
Few apps on android even have good animations because of this I think.
* Fixes white screen between splash screen and app screen on iOS.
* Fixes sign up layout significantly (replacing weird layout stuff with flex + better image sizing)

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

* Galaxy S10 device
* Pixel 2 device
* Pixel 6 simulator
* iOS simulator
* @sliptype please test this after it goes in on the one plus device!

Status bar changes yield great results on android

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-1602

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-1602

File: README.md [hunk 1]
```diff
@@ -62,14 +62,23 @@ npm run desktop:dev
...
+xcrun xctrace list devices
+npm run ios:<env> -- --device "My iPhone"
 
+# android
 npm run android:dev
 npm run android:stage
 npm run android:prod
+# on a physical device
+adb devices
+npm run android:<env> -- --deviceId "A38M608KHBK"
```
Comment (sliptype) [inline README.md:81]: Btw I'm able to run on device with `npm run android:prod -- --device`

File: packages/web/src/common/store/backend/sagas.ts [hunk 2]
```diff
@@ -82,6 +82,7 @@ export function* setupBackend() {
   yield* put(accountActions.fetchAccount())
   const { web3Error, libsError } = yield* call(audiusBackendInstance.setup)
   if (libsError) {
+    alert('OH NO')
```
Comment (sliptype) [inline packages/web/src/common/store/backend/sagas.ts:85]: 👀

**Review by sliptype — APPROVED** [review]
Amazing! Thanks for doing this, let's make sure to test on multiple screen sizes via the emulator

**Review by dylanjeffers — APPROVED** [review]

**Comment by RamProg** [comment]
We are having the same issue. Looks good on an emulated Pixel 5, then too small on a physical Samsung Galaxy S20+

**Pull request merged**